api/src/main/java/org/pmiops/workbench/dataset/mapper/DataSetMapper.java

```diff
@@ -96,13 +96,19 @@ default void populateFromSourceDbObject(
 
   default List<PrePackagedConceptSetEnum> prePackagedConceptSetFromStorage(
       List<Short> prePackagedConceptSet) {
+    if (prePackagedConceptSet == null) {
```

Comment:
`NullValueCheckStrategy.ALWAYS` should handle this already - is this not working?

Reply:
Thanks for the pointer, I had not seen this.

It seems the only place this is applied currently is on the etag/version conversion (in fact, that's the only reference to this property I can find in the codebase). Is there a way to specify this for all fields in the mapper, or is there something I'm missing here where we'd expect that to be applied in this case?

Reply:
sorry, I don't really know much about this.  Is it handling nulls by returning empty list instead?